### PR TITLE
feat(README): Internal Resources for help

### LINKED ISSUE #273: Need Help? section in README
**Is your feature request related to a problem? Please describe.**
Based on a team meeting discussion. We should encourage developers to get feedback and help from blockers sooner rather than later. 

**Describe the solution you'd like**
The Main README should have a section on help and direct developer to our technical feedback slack channel as well as another list of training slacks for specific stacks in the organization (for example training-Django). We discussed that if developers are spending more than 30-45 minutes on a task and are still stuck, there should be a good place to discuss technical issues. This should also be explicit within our S&P as to be useful for someone coming into the organization.

### PR DESCRIPTION
**Pull request by michaelachrisco**
## Changes
1. Add in slack channels and other references for help with blockers.

## Purpose
Make developers aware of our slack channels that can help them out. 

## Approach
Add in the start of the basic slack channels that we have available within the org. 

## Learning
If anyone knows of some good BW specific slack channels, resources, outreach comment below and we can add it to the list! I can see this becoming another page eventually. 

Closes #273

### CODE REVIEW

**Review by Basim-Elayan — APPROVED** [review]
Nice addition and information. might we could later mention looking to our stack overflow account.

**Comment by michaelachrisco** [comment]
> Nice addition and information. might we could later mention looking to our stack overflow account.

Oh thats a good one! I didn't think of that, thanks @Basim-Elayan !  

Maybe a direct link over to our specific one? Do we happen to know if anyone outside the org can sign up for that? I dont think we want to include the link if we have direct registration enabled.

**Comment by Basim-Elayan** [comment]
> > Nice addition and information. might we could later mention looking to our stack overflow account.
> 
> Oh thats a good one! I didn't think of that, thanks @Basim-Elayan !
> 
> Maybe a direct link over to our specific one? Do we happen to know if anyone outside the org can sign up for that? I dont think we want to include the link if we have direct registration enabled.

Good question, I'm not sure might we can check that out.

**Review by justincohan — APPROVED** [review]

**Review by FlavioShift3 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -102,6 +102,13 @@ Using Chocolatey has several advantages:
 
 - If your job entails design work, you will need to get access to the Bitwise Adobe Photoshop license.
 
+## Need Help?
+Getting stuck is a natual part of development and happends to the best of us. Its better to reach out than to stay silent and we encourage you to do so. If you find yourself stuck on a task for more than 45 minutes, here are a couple of places to get help:
```
[the remark this answers is not in the document]
Comment (michaelachrisco) [reply README.md]: Fixed via: 9b6ac6382e74353b734a2c7069de15ac12effa0a

Thanks!

**Comment by michaelachrisco** [comment]
Thanks everyone!

**Pull request merged**